tests/sentry/web/api/tests.py

```diff
@@ -41,13 +46,17 @@ def test_missing_csp_report(self):
         )
         assert resp.status_code == 400, resp.content
 
+    @pytest.mark.obsolete(
+        "ported to Relay", "tests/integration/test_security_report.py::test_uses_origins"
+    )
     @mock.patch("sentry.utils.http.get_origins")
     def test_bad_origin(self, get_origins):
         get_origins.return_value = ["example.com"]
         resp = self.client.post(
             self.path,
             content_type="application/csp-report",
-            data='{"csp-report":{"document-uri":"http://lolnope.com","effective-directive":"img-src","violated-directive":"img-src","source-file":"test.html"}}',
+            data='{"csp-report":{"document-uri":"http://lolnope.com","effective-directive":"img-src",'
```

Comment:
_nit_: Unintended reformat?

Reply:
This was an intended reformat (the line is too long, over 160 columns).